### PR TITLE
ref(tsc): convert User,Member and AccessRequest fixtures to tsx

### PR DESCRIPTION
**Pull request by JonasBa**

### CODE REVIEW

File: fixtures/js-stubs/accessRequest.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import {Team} from './team';
+
+export function AccessRequest(
+  params: Partial<AccessRequestType> = {}
+): AccessRequestType {
+  return {
+    id: '123',
+    member: Member(),
+    team: Team(),
+    requester: params.requester ?? {
```
Comment (ryan953) [inline fixtures/js-stubs/accessRequest.tsx:13]: I think that `params.requester ??` will get overridden by the spread below

Comment (JonasBa) [reply fixtures/js-stubs/accessRequest.tsx:13]: ah yeah, will change

**Comment by JonasBa** [comment]
requires https://github.com/getsentry/getsentry/pull/11058

File: fixtures/js-stubs/accessRequest.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {AccessRequest as AccessRequestType} from 'sentry/types';
+
+import {Member} from './member';
+import {Team} from './team';
+
+export function AccessRequest(
+  params: Partial<AccessRequestType> = {}
+): AccessRequestType {
```
Comment (ryan953) [inline fixtures/js-stubs/accessRequest.tsx:8]: We're also going to need to update https://github.com/getsentry/sentry/blob/299d07e193818163ef85f0f42c7020744b873014/fixtures/js-stubs/types.tsx#L23

Comment (ryan953) [reply fixtures/js-stubs/accessRequest.tsx:8]: i'm happy to see that be separate again, i'm worried it's a big can of worms

**Review by ryan953 — APPROVED** [review]

File: fixtures/js-stubs/accessRequest.tsx (hunk 2 above)
Comment (JonasBa) [reply fixtures/js-stubs/accessRequest.tsx:8]: Ah yes, one sec

Comment (JonasBa) [reply fixtures/js-stubs/accessRequest.tsx:8]: Type inference for stubs will def need to be fixed/improved, but we need to move them to TS first

**Comment by JonasBa** [comment]
@ryan953 mind reviewing https://github.com/getsentry/getsentry/pull/11058 as well so we can merge this?

**Pull request merged**